packages/wrangler/src/utils/create-batches.ts

```diff
@@ -0,0 +1,8 @@
+export function* createBatches<T>(
```

Comment:
Is there a reason this needs to be a generator rather than a standard function returning an array?